examples/regex.yml

```diff
@@ -18,7 +18,7 @@ commands:
     cmd: "666"
     mode: findall
     output:
-        # {'MATCH_0': '6', 'MATCH_1': '6', 'MATCH_2': '6'}
+        # {'MATCH_0': '666'}
```

Comment:
The expected result is: `{'MATCH_0': '6', 'MATCH_1': '6', 'MATCH_2': '6'}`

Reply:
@whotwagner , I actually changed this per your previous review, 
![image](https://github.com/user-attachments/assets/d52af73c-138b-4d60-9423-9b34b1c0945e)
 i think 666 is the correct match? (if the findall would look for "6" only we would expect 3 results)

Reply:
Sorry, I made a mistake back then. The correct result would be 3 matches:

```
Python 3.12.3 (main, Apr 10 2024, 05:33:47) [GCC 13.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> var = "666"
>>> import re
>>> re.findall("6", var)
['6', '6', '6']
```

Reply:
But if it's `cmd: "666"` then it's doing a `re.findall("666", var)`, or?

Reply:
> But if it's `cmd: "666"` then it's doing a `re.findall("666", var)`, or?

true. sorry, my bad (again)